app/controllers/application.js

```diff
@@ -1,15 +1,30 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 import { set } from "@ember/object";
-import { reads } from '@ember/object/computed';
+import { /*, printSprites */ } from 'ember-animated';
+import move from 'ember-animated/motions/move';
+import { easeInAndOut } from 'ember-animated/easings/cosine';
 
 export default Controller.extend({
   dj             : service(),
   hifi           : service(),
   currentStream  : service(),
   links: [ { 'href': null, 'nav-slug': 'listen', 'title': 'Listen'}, { 'href': null, 'nav-slug': 'playlist-history', 'title': 'Playlist History'} ],
 
-  showPlayer: reads('dj.showPlayer'),
+  showPlayer: true,
+
+  * showPlayerAnimation(context) { //eslint-disable-line
+    // printSprites(context, 'placeholder')
```

Comment:
delete this (along with import of `printSprites`) if you aren't using it anymore (is this a debug statement?)